lib/form_data.js

```diff
@@ -235,7 +235,12 @@ FormData.prototype._getContentDisposition = function(value, options) {
   }
 
   if (filename) {
-    contentDisposition = 'filename="' + filename + '"';
+    // Escaping to prevent CRLF Injection / tampering with invalid fields.
+    // https://html.spec.whatwg.org/#multipart-form-data
+    var escapedFilename = filename.replaceAll('"', '%22')
+      .replaceAll('\r', '%0D')
+      .replaceAll('\n', '%0A');
```

Comment:
v4 supports down to node 6, and node 6 doesn't have .replaceAll. However, I believe you can use this:
```suggestion
    var escapedFilename = filename.replaceAll('"', '%22')
      .replace(/\r/g, '%0D')
      .replace(/\n/g, '%0A');
```